src/game/RandomizationOfShipsPositions.js

```diff
@@ -0,0 +1,163 @@
+import { SHIPS_LIST } from "./constants";
+
+const GAME_BOARD = document.querySelectorAll(".board__fields");
+
+let gameBoardArray = [];
+let fieldsNamesArr = [];
+let direction;
+let position;
+const GAME_BOARD_WIDTH = 10;
+
+const createFieldsName = () => {
+  for (let i = 0; i < 10; i++) {
+    for (let j = 0; j < 10; j++) {
+      fieldsNamesArr.push(`${String.fromCharCode(i + 65)}${j + 1}`);
+    }
+  }
+};
+createFieldsName();
+
+const createGameBoardArray = () => {
+  gameBoardArray = [];
+  for (let i = 0; i < GAME_BOARD_WIDTH; i++) {
+    gameBoardArray.push([]);
+    for (let j = 0; j < GAME_BOARD_WIDTH; j++) {
+      gameBoardArray[i].push({
+        isEmpty: true,
+        shipName: "",
+      });
+    }
+  }
+};
+
+const directionRoll = () => {
+  Math.floor(Math.random() * 2) === 0
+    ? (direction = "vertical")
+    : (direction = "horizontal");
+};
+
+const makePosition = (shipIndex) => {
+  position = [];
+  for (let i = 0; i < 2; i++) {
+    position.push(Math.floor(Math.random() * GAME_BOARD_WIDTH));
+  }
+  if (direction === "horizontal") {
+    if (
+      Number(`${position[1]}`) + SHIPS_LIST[shipIndex].size >
```

Comment:
What is the point of using here `Number` constructor and `string literal`?

Reply:
From what I see here you can make it simpler
`	if (
		direction === 'horizontal' &&
		position[1] + SHIPS_LIST[shipIndex].size > GAME_BOARD_WIDTH
	) {
		position[1] = Math.abs(position[1] - SHIPS_LIST[shipIndex].size);
	} else if (position[0] + SHIPS_LIST[shipIndex].size > GAME_BOARD_WIDTH) {
		position[0] = Math.abs(position[0] - SHIPS_LIST[shipIndex].size);
	}`